database/src/main/resources/db/migration/V1.84.0__RHCLOUD-25882_add_indexes_on_notification_drawer.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX ix_drawer_notification_org_id_user_id_created_read_event_id ON drawer_notification (org_id, user_id, created, read, event_id);
```

Comment:
Will drawer entries always be sorted in descending order? If so, it could help to add DESC to the index declaration:
```suggestion
CREATE INDEX ix_drawer_notification_org_id_user_id_created_read_event_id ON drawer_notification (org_id, user_id, created DESC, read, event_id);
```

Reply:
Good point